File: examples/comparison/strings/concat/bench.rs
fn main() {
    const MOD: i64 = 1000000007;
    let mut s = String::from("a");
    let mut acc: i64 = 0;
    for _ in 0..100000 {
        s.push('x');
        acc = (acc + s.len() as i64).rem_euclid(MOD);
    }
    println!("{}", acc);
}

File: examples/comparison/strings/concat/bench.go
package main

import (
	"fmt"
	"runtime"
)


func main() {

	const mod = 1000000007
	var acc int64 = 0
	s := "a"
	for i := 0; i < 100000; i++ {
		s += "x"
		acc = (acc + int64(len(s))) % mod
	}
	runtime.KeepAlive(acc)
	fmt.Println(acc)
}
